themes/hugo-steam-theme/layouts/partials/head.html

```diff
@@ -19,6 +19,17 @@
       <link href="{{ .RSSLink }}" rel="feed" type="application/rss+xml" title="{{ .Site.Title }}" />
     {{ end }}
 
+    <meta property="og:type" content="article" />
+    <meta property="og:title" content="{{ .Title }}"/>
+    <meta property="og:description" content="{{ .Description | markdownify }}" />
+    <meta name="og:image" content="https:{{ .Site.BaseURL }}{{.Params.image}}" />
```

Comment:
If people share a listing, the image will be empty. We could maybe fallback in our logo, couldn't we?

Reply:
fixed